scripts/migrate_tools.py

```diff
@@ -10,33 +10,8 @@ def deprecated_tool():
 
 
 orgs = OrganizationManager().list_organizations(cursor=None, limit=5000)
-for org in orgs:
+for org in tqdm(orgs):
```

Comment:
Why do we want tqdm?

Reply:
(y)

Reply:
discussed offline, just for visual purposes while running this script on pod